src/components/Partners.astro

```diff
@@ -54,6 +54,10 @@ import HeadingFlanked from "./HeadingFlanked.astro";
 		max-width: 90vw;
 	}
 
+	.partners-blurb {
+		margin: 0 auto;
+	}
+
```

Comment:
There's just a _bit_ of viewport width range that still overflows. I'll fix that up real quick...

![Screenshot of the three partners with a bit of horizontal overflow, around 900px viewport width](https://github.com/user-attachments/assets/25c5e049-c47f-4e87-ae0f-bf6de836f356)

Reply:
Thanks for fixing this up!